Date: Dec 7, 2015 12:24 PM

From: a43ec06a9f9db3c3952fe6c2cfa1acbfc267bdde Dec 7, 2015 12:36 PM

Subject: [PATCH] RDKB-3168: uclibc: Backport from latest OE master

	Try to fix clone() related getpid bugs and posix_spawn.
	
Source: OPENEMBEDDED


Upstream-Status: Downloaded  [https://www.openembedded.org/pipermail/openembedded-core/2015-August/227297.html] 

Signed-off-by: Khem Raj <raj.khem at gmail.com>
---

Index: git/libc/sysdeps/linux/i386/bits/fcntl.h
===================================================================
--- git.orig/libc/sysdeps/linux/i386/bits/fcntl.h
+++ git/libc/sysdeps/linux/i386/bits/fcntl.h
@@ -50,6 +50,7 @@
 # define O_NOFOLLOW	0400000	/* Do not follow links.	 */
 # define O_NOATIME     01000000 /* Do not set atime.  */
 # define O_CLOEXEC     02000000 /* Set close_on_exec.  */
+# define O_PATH       010000000 /* Resolve pathname but do not open file.  */
 #endif
 
 /* For now Linux has synchronisity options for data and read operations.
